node_modules/gh-rest/lib/rest/admin.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (c) 2015 "Fronteer LTD"
+ * Grasshopper Event Engine
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+module.exports = function(client) {
+    client.admin = {};
+
+    /**
+     * Get a list of global administrators
+     *
+     * @param  {Object}             [opts]                  Optional properties
+     * @param  {Number}             [opts.limit]            The maximum number of results to retrieve. Default: 10
+     * @param  {Number}             [opts.offset]           The paging number of the results to retrieve
+     * @param  {Function}           callback                Standard callback function
+     * @param  {Object}             callback.err            An error that occurred, if any
+     * @param  {GlobalAdminList}    callback.globalAdmins   The requested list of global administrators
```

Comment:
FWIW, for the gh-rest we usually document the callback parameters like so:
```
     * @param  {Function}       callback                Standard callback function
     * @param  {Object}         callback.err            An error that occurred, if any
     * @param  {Object}         callback.body           The JSON response from the REST API
     * @param  {Response}       callback.response       The response object as returned by requestjs
     * @see gh-admins/lib/rest.js for more information
```

No need to change these though